docs/src/pages/system/palette/BackgroundColor.tsx

```diff
@@ -6,47 +6,61 @@ export default function BackgroundColor() {
   return (
     <Grid container spacing={1}>
       <Grid item xs={12} sm={4}>
-        <Box bgcolor="primary.main" color="primary.contrastText" p={2}>
+        <Box
+          sx={{ bgcolor: 'primary.main', color: 'primary.contrastText', p: 2 }}
+        >
```

Comment:
?

```suggestion
        <Box sx={{ bgcolor: 'primary.main', color: 'primary.contrastText', p: 2 }}>
```

Reply:
Will check if I can change it, I think it's prettier.

Reply:
Ok, I wouldn't be surprised, we allow fewer char per ligne for the demos of the documentation.

Reply:
Yep, it's prettier, will have to leave it as is.